pycalphad/core/hyperplane.pyx

```diff
@@ -84,6 +84,12 @@ cpdef double hyperplane(double[:,::1] compositions,
     Examples
     --------
     None yet.
+
+    Notes
+    -----
+    M: number of energy points that have been sampled
+    N: number of components
+    P: N+2, max phases by gibbs phase rule
```

Comment:
Needs to be changed to N+1. Maybe choose another letter besides P to represent this quantity, so it isn't easy to confuse with a thermodynamic variable

Reply:
(though we do already use N, so...)

Reply:
I think defining them prevents confusion for overloading (but not having any labels was confusing).